src/js/utilities/characterFilters.js

```diff
@@ -0,0 +1,9 @@
+/*  getFilteredCharactersByAlignment
+*   @param {[Character]} Character array to filter
+*   @param {String}   Alignment of characters in output arr
+*/
+
+export function getFilteredCharactersByAlignment(characters, alignment) {
+  const result = characters.filter((char) => char.biography.alignment === alignment);
+  return result;
```

Comment:
Declaring a variable here is unnecessary. This can be simplified as:
```suggestion
  return characters.filter((char) => char.biography.alignment === alignment);
```